openmrs/src/test/java/org/motechproject/openmrs/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -282,18 +282,32 @@ public void shouldCreateProgramEnrollment() {
         DateTime dateEnrolled = new DateTime("2000-08-16T07:22:05Z");
         DateTime dateCompleted = new DateTime("2100-08-16T07:22:05Z");
 
+        Map<String, String> programEnrollmentAttributes = new HashMap<>();
+        programEnrollmentAttributes.put("8d8718c2-c2cc-11de-8d13-0010c6dffd0f", "testValue");
+
+        List<Attribute> attributes = new ArrayList<>();
+        Attribute attribute = new Attribute();
+        Attribute.AttributeType attributeType = new Attribute.AttributeType();
+
+        attributeType.setUuid("8d8718c2-c2cc-11de-8d13-0010c6dffd0f");
+        attribute.setValue("testValue");
+        attribute.setAttributeType(attributeType);
+
+        attributes.add(attribute);
+
         ProgramEnrollment programEnrollment = new ProgramEnrollment();
 
         programEnrollment.setPatient(patient);
         programEnrollment.setProgram(program);
         programEnrollment.setDateEnrolled(dateEnrolled.toDate());
         programEnrollment.setDateCompleted(dateCompleted.toDate());
         programEnrollment.setLocation(location);
+        programEnrollment.setAttributes(attributes);
 
         doReturn(Collections.singletonList(location)).when(locationService).getLocations(eq(CONFIG_NAME), eq(location.getName()));
 
         openMRSActionProxyService.createProgramEnrollment(CONFIG_NAME, patient.getUuid(), program.getUuid(),
-                dateEnrolled, dateCompleted, location.getName());
+                dateEnrolled, dateCompleted, location.getName(), programEnrollmentAttributes);
 
         verify(programEnrollmentService).createProgramEnrollment(eq(CONFIG_NAME), programEnrollmentCaptor.capture());
         assertEquals(programEnrollment, programEnrollmentCaptor.getValue());
```

Comment:
ProgramEnrollment does not have attributes in equals method, so this check does not verify attributes